### ISSUE TITLE
Video Autoplay + Sound On for Lightbox

### ISSUE THREAD

**Issue by maxproway** [soflyy/breakdance-bugs#831]
Hi, I'm trying to use lightbox video as pop up.

When I using vimeo - there no sound and no autoplay.
YouTube - no sound but have autoplay. 

Adding ?autoplay=1&mute=0 doesn't help, since the system automatically adds **?autoplay=0&mute=1** to the embed iframe.

I don't know is it bug (because users have to click 3 times to lunch the video: pop up trigger, unmute a sound and click a play button) or its a feature.
In the case of the second could you please add a settings to lightbox video content to have an ability to  set autoplay and sound.

Thank you!

**Comment by jeffmitchdotcom** [comment]
Yes, please! If the user is pressing a button to launch a video, it should minimally not have it be muted by default with no way to override. I can understand wanting to avoid autoplaying videos with sound on a page when nothing is done by the user to trigger a video -- but even that should be at the discretion of the web designer to build a positive experience for their visitors.

Adding manual settings for the Button > Link > Lightbox > Video Type for AutoPlay and Mute would be amazing!

**Comment by breakdancemoderator** [comment]
The issue is with the lightGallery module we are using https://github.com/sachinchoolur/lightGallery/issues/1378.

We remain hopeful for this to be addressed in that module, but ultimately trust the author's judgement. We are open to patching / forking the lightGallery module if the original author deems the issue invalid.

However, and perhaps more importantly, browser vendors / users tend not to like autoplay w/ sound on and fighting them on this tends to be a losing battle.

This issue will remain open.

**Comment by mhunt** [comment]
For anyone looking for a workaround in breakdance, and you're using a button to open the video, you can simply create a pop-up template and place a video block in there, then for your button action, select a pop-up instead of a lightbox.
